src/main/java/org/jenkinsci/plugins/envinject/EnvInjectBuildWrapper.java

```diff
@@ -104,8 +101,8 @@ public void buildEnvVars(Map<String, String> env) {
                     env.putAll(resultVariables);
                 }
             };
-        } catch (Throwable throwable) {
-            logger.error("Problems occurs on injecting env vars as a build wrap: " + throwable.getCause());
+        } catch (Exception e) {
```

Comment:
I'm not sure this is a right change. Groovy engine may case many issues like OOM, and this change causes unrecoverable build failure